app/javascript/src/components/Dashboard/Notes/Note/Tooltip.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+import { Button } from "neetoui";
+
+const Tooltip = () => (
+  <Button
+    className="absolute right-12"
+    label="Wednesday 10:30AM"
+    onClick={() => {}}
+  />
+);
```

Comment:
- The component name and the content doesn't match
- More than that this abstraction is pretty unnecessary. The content in this component doesn't justify the need for making it a separate component. It's very small and not too complex.
- Unless this component is reused at multiple places, it doesn't make sense to have this as a separate component.

Reply:
Removed this component.